Source/Renderer/SFMLRenderer.cpp

```diff
@@ -6,29 +6,4 @@
 void SFMLRenderer::add(const sf::Drawable &drawable)
 {
     m_draws.push_back(&drawable);
-}
-
-void SFMLRenderer::render(sf::Window &window)
-{
-    /*
-    if (m_draws.empty())
-        return;
-
-    glDisable(GL_DEPTH_TEST);
-    glBindBuffer(GL_ARRAY_BUFFER, 0);
-    glBindVertexArray(0);
-    glBindTexture(GL_TEXTURE_2D, 0);
-    glBindTexture(GL_TEXTURE_CUBE_MAP, 0);
-    glUseProgram(0);
-
-    window.pushGLStates();
-    window.resetGLStates();
-
-    for (const auto& draw : m_draws)
-    {
-        window.draw(*draw);
-    }
-
-    window.popGLStates();
-    m_draws.clear();*/
-}
```

Comment:
As the SFML renderer isn't used anymore, it could be fully remove instead of commented out

I think this is because when I used windows, it worked, but on Linux, it did not.

This is because of the way SFML changes the OpenGL state and uses older OpenGL version that aren't compatible with OpenGL 3.0 + 

It's surprising it worked on windows in the first place.

Reply:
Completely understandable. This should be a relatively quick fix.